README.md

```diff
@@ -80,39 +85,6 @@ android {
 }
 ```
 
-### AWS Mobile Client
```

Comment:
Should we instead include a note about setting up Auth, before API and DataStore will work? It could still be a worthwhile call out. The use of a disparate library to do auth was a surprise in preview. The need to use Auth is less of a surprise, but still not immediately obvious to a new comer, I'd suppose.

Reply:
Yes, I think that's a good idea! I'll write something up.

Reply:
Wrote something tiny.